packages/headless/src/controllers/search-box/headless-search-box.ts

```diff
@@ -157,12 +167,16 @@ export function buildSearchBox(
   };
 }
 
-function getSuggestions(state: QuerySuggestState | undefined) {
+function getSuggestions(
+  state: QuerySuggestState | undefined,
+  highlightOptions: SuggestionHighlightingOptions
+) {
   if (!state) {
     return [];
   }
 
   return state.completions.map((completion) => ({
-    value: completion.expression,
+    value: getHighlightedSuggestions(completion.highlighted, highlightOptions),
+    rawValue: completion.expression,
```

Comment:
This is used to set the attribute `value` on the suggestion in atomic. I think this is useful if a user wants to use the non-highlighted value while still having the delimiters set when building the search box. Otherwise, the user would have to parse the string to remove the delimiters to get to the raw value.

Reply:
I am unsure about the naming, is `rawValue` the one with the delimiters, or without them?

Reply:
I like those renames 💯